pytorch_to_returnn/import_wrapper/context.py

```diff
@@ -118,12 +121,28 @@ def _raise_not_implemented(*args):
     torch.nn.Module: ExplicitWrappedType(torch.nn.Module, WrappedModuleBase, wrap=_raise_not_implemented),
   }
 
+  class Functional(WrappedModuleBase):
```

Comment:
I think I would move this out somewhere. Maybe to `base_wrappers.py`.

Reply:
You mean `base_wrappers/function.py`? Or maybe somewhere in `torch_wrappers/`?

Reply:
Yea maybe `torch_wrappers/function.py`.